### DISCUSSION TITLE
Version number potentially not bumped from 7.2.0 to 7.3.0?

### DISCUSSION THREAD

**Discussion by ghost** [GoogleChrome/lighthouse#12406] [Q&A]
**Summary**

Might just be a configuration error on my end but when I checked my lighthouse version it prompted me to update from 7.2.0 to 7.3.0, after doing so, I only get given the version number 7.2.0 after re-running lighthouse --version (the prompt asking to update also disappeared).

`lighthouse --version`: 7.2.0
`npm view lighthouse version`: 7.3.0
`yarn why lighthouse`: 7.3.0

Lighthouse generated reports also say 7.2.0, any way I can confirm which version is installed?

**Comment by patrickhulce** [comment]
Yes, the [version number was correctly bumped](https://unpkg.com/lighthouse@7.3.0/package.json), this sounds like a global install vs. local install issue. 

I cannot reproduce the issue you're talking about.

```bash
mkdir test-lighthouse
cd test-lighthouse

npm i lighthouse
./node_modules/.bin/lighthouse --version
> 7.3.0
```

Double check `which lighthouse` vs. `./node_modules/.bin/lighthouse`?
